### ISSUE TITLE
Code 65 compile error on Xcode 12 beta 4

### ISSUE THREAD

**Issue by psycrtb** [fiskaly/fiskaly-sdk-swift#29]
## Bug Report
We are having issues with a code 65 error on the Xcode 12 beta 4 compiler. This applies to fiskaly-sdk 1.2.100 19th aug release.

This is specific to this selected version. And switching back to xcode 11.6 causes no issues.

1: The latest xcode beta is required and must be installed in parallel to your current built.
2: The xcode command line tools must be installed using the beta
3: The beta tools must be activated  (Xcode->Preferences->Locations->Command line tools) and set to Xcode 12.0
4: Attempt a standard install using Carthage.

A dump of the following output is the response from xcode build tools:
`tim@yeah ios-app % carthage update
*** Fetching fiskaly-sdk-swift
*** Checking out fiskaly-sdk-swift at "v1.2.100"
*** xcodebuild output can be found in /var/folders/x3/g_2w1l797cx_m55h3zsdq1qh0000gp/T/carthage-xcodebuild.7ZIvjE.log
*** Downloading fiskaly-sdk-swift.framework binary at "Release v1.2.100"
*** Building scheme "FiskalySDK" in FiskalySDK.xcodeproj
Build Failed
	Task failed with exit code 65:
	/usr/bin/xcrun xcodebuild -project /Users/tim/Documents/r2oSource/ios-app/Carthage/Checkouts/fiskaly-sdk-swift/FiskalySDK.xcodeproj -scheme FiskalySDK -configuration Release -derivedDataPath /Users/tim/Library/Caches/org.carthage.CarthageKit/DerivedData/12.0_12A8179i/fiskaly-sdk-swift/v1.2.100 -sdk iphonesimulator -destination platform=iOS\ Simulator,id=1DE7A65B-DE3C-4E62-9C63-99C2DD2F7FF8 -destination-timeout 3 ONLY_ACTIVE_ARCH=NO CODE_SIGNING_REQUIRED=NO CODE_SIGN_IDENTITY= CARTHAGE=YES build (launched in /Users/tim/Documents/r2oSource/ios-app/Carthage/Checkouts/fiskaly-sdk-swift)

This usually indicates that project itself failed to compile. Please check the xcodebuild log for more details: /var/folders/x3/g_2w1l797cx_m55h3zsdq1qh0000gp/T/carthage-xcodebuild.7ZIvjE.log`

Note that when i unset the beta tools and switch back (Xcode->Preferences->Locations->Command line tools) and set to Xcode 11.6 command line tools it builds correctly.

We do currently deploy our code to customers using the 11.6 toolset but test internally using 12.0 so this isn't a massive issue for us. So xcode 12 is only used dog food our app and tools at this time of year in order to reduce the QA pressure on iOS major version updates. We thought we should share this with you.

**Comment by prempador** [comment]
Hello @psycrtb, 

I have done some research and saw a lot of issues regarding Xcode 12 Beta and Carthage, where people experience similar issues. 
For now this is out of scope for me to try fixing but I will come back to this issue as soon as I have some spare time. 

Best,
@prempador

**Comment by psycrtb** [comment]
Hi guys,
Is this related to the carthage issue: https://github.com/Carthage/Carthage/issues/3019
If you think it is it may be worth closing and then pinning this ticket.

**Comment by JohannesFerner** [comment]
@psycrtb is this still happening?

**Comment by psycrtb** [comment]
Hi @JohannesFerner. 
I am unable to answer that question definitively because I now no longer work on the project using this framework. 

I would probably suggest you close this ticket and wait for further information should it appear.
